app/controllers/api/group_assignments_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module API
+  class GroupAssignmentsController < API::ApplicationController
+    include ActionController::Serialization
+    include OrganizationAuthorization
+
+    before_action :set_assignment, except: :index
```

Comment:
Nitpick: instead of `except: :index` you could use `only: :show` to avoid any unwanted side effects of adding a new route later